app.js

```diff
@@ -0,0 +1,17 @@
+const express = require('express');
+const mongoose = require('mongoose');
+const bodyParser = require('body-parser');
+const commentRoutes = require('./routes/comments');
+
+const app = express();
+
+mongoose.connect('mongodb://localhost:27017/mydb', { useNewUrlParser: true, useUnifiedTopology: true });
+
+app.use(bodyParser.json());
+
+app.use('/', commentRoutes);
```

Comment:
I think it would be better if you will write /comments here and then in the comments routes it would look different